airflow/www/templates/airflow/tree.html

```diff
@@ -102,10 +112,63 @@
 
 var tree = d3.layout.tree().nodeSize([0, 25]);
 var nodes = tree.nodes(data);
+var now_ts = Date.now()/1000;
 var nodeobj = {};
 for (i=0; i<nodes.length; i++) {
```

Comment:
perhaps we could stick all this node munging code into a function?

Reply:
good call, refactored the loop into `backfill_taskinstance_properties` function.